zerosum/zerosum.py

```diff
@@ -272,7 +272,7 @@ def zerosum(entries, options_map, config):
     # TODO: should ideally track account specific earliest date
     new_open_entries = create_open_directives(new_accounts, entries)
 
-    print("Zerosum: {}/{} postings matched. {} multiple matches. {} new accounts added.".format(match_count, zerosum_postings_count, multiple_match_count, len(new_open_entries)))
+    #print("Zerosum: {}/{} postings matched. {} multiple matches. {} new accounts added.".format(match_count, zerosum_postings_count, multiple_match_count, len(new_open_entries)))
```

Comment:
Stats are helpful. I agree printing may not be the best idea. Can we log this instead of commenting this out?

Reply:
The reason I comment it out is that this stdout triggers error in nvim autocomplete plugin. Do you know if there's any way to pass the stats down in the Importer protocol?

Reply:
I don't readily see a way to do that, but let me at least make this conditional. Checking this in in a couple minutes.

Reply:
Done in abeac3b